pkg/services/object/get/v2/util.go

```diff
@@ -646,3 +677,18 @@ func checkStatus(stV2 *status.Status) error {
 
 	return nil
 }
+
+func chunkToSend(global, local int, chunk []byte) []byte {
+	if global == 0 || global == local {
```

Comment:
What's `global == 0` condition for?

Reply:
if `global == 0` then `local == 0` too, agree. it was the first line i wrote but then i decided that it makes the func more explicit

dropped